runtime/ops/tty.rs

```diff
@@ -118,6 +118,30 @@ fn op_stdin_set_raw(
   }
   #[cfg(unix)]
   {
+    fn prepare_stdio() {
```

Comment:
Can `op_stdin_set_raw` be called more than once? Should this be in a OnceCell?

Reply:
Oopsie, nice catch